docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -28,6 +29,256 @@
+		* Removed property: `hormonal_contraceptive_type`
+		* Removed property: `hormonal_contraceptive_use`
+		* Removed property: `hpv_positive_type`
+		* Removed property: `hysterectomy_margins_involved`
+		* Removed property: `hysterectomy_type`
+		* Removed property: `immunosuppressive_treatment_type`
+		* Removed property: `menopause_status`
+		* Removed property: `nadir_cd4_count`
+		* Removed property: `pancreatitis_onset_year`
+		* Removed property: `pregnancy_count`
+		* Removed property: `pregnancy_outcome`
+		* Removed property: `reflux_treatment_type`
+		* Removed property: `risk_factor`
+		* Removed property: `risk_factor_method_of_diagnosis`
+		* Removed property: `risk_factor_treatment`
+		* Removed property: `risk_factors`
+		* Removed property: `undescended_testis_corrected`
+		* Removed property: `undescended_testis_corrected_age`
+		* Removed property: `undescended_testis_corrected_laterality`
+		* Removed property: `undescended_testis_corrected_method`
+		* Removed property: `undescended_testis_history`
+		* Removed property: `undescended_testis_history_laterality`
+		* Removed property: `viral_hepatitis_serologies`
+		* Removed property: `weight`
+		* Changes made to `days_to_first_event`
+		* Changes made to `timepoint_category`
+			* New permissible value: `After Study Registration`
+* Altered `read_group` Entity
+	* Changes made to `deprecated`
+		* Removed permissible value: `RIN`
+	* Changes made to `properties`
+		* Changes made to `single_cell_library`
+			* New permissible value: `Chromium 3' Gene Expression v1 Library`
+			* New permissible value: `Chromium 3' Gene Expression v4 Library`
+			* New permissible value: `Chromium Next GEM Single Cell Multiome ATAC + Gene Expression Library`
+		* Changes made to `target_capture_kit`
+			* New permissible value: `xGen Exome Research Panel v2.0`
+* Altered `sample` Entity
+	* Changes made to `deprecated`
+		* New permissible value: `tumor_code_id`
+	* Changes made to `properties`
+		* Removed property: `composition`
+		* Removed property: `is_ffpe`
+		* Removed property: `oct_embedded`
+		* Removed property: `sample_type_id`
+		* Removed property: `tumor_code`
+		* Changes made to `biospecimen_anatomic_site`
+			* Removed permissible value: `Brain Stem`
+			* Removed permissible value: `Common Duct`
+			* Changes made to `deprecated_enum`
+				* Removed permissible value: `Brain Stem`
+				* Removed permissible value: `Common Duct`
+		* Changes made to `method_of_sample_procurement`
+			* Removed permissible value: `Indeterminant`
+			* Removed property: `deprecated_enum`
+* Altered `analyte` Entity
+	* Changes made to `deprecated`
+		* New permissible value: `ribosomal_rna_28s_16s_ratio`
+	* Removed permissible value: `analyte_type_id`
+	* Changes made to `properties`
+		* New property: `dna_integrity_number`
+		* New property: `ribosomal_rna_28s_18s_ratio`
+		* Removed property: `analyte_type_id`
+* Altered `exposure` Entity
+	* Removed property: `deprecated`
```

Comment:
another slightly odd deprecated one.

Reply:
Yeah, let's remove it.  I think it's an artifact of removing all of the deprecated props, so the whole "deprecated" section gets removed.